### ISSUE TITLE
Tracks in BBEditor blanks out

### ISSUE THREAD

**Issue by zonkmachine** [LMMS/lmms#2957]
_Version 1.1.90-g6f317ac (Linux/x86_64, Qt 4.8.6, GCC 4.8.4)_
BBEditor is turning up with blank tracks, sometimes all tracks blank and sometimes just a couple of them. I've seen no clear patterns so far but you can clone/add a BBTrack with just the original Kicker and no notes selected. Song paused, just cloning and deleting, and eventually blank tracks will show up in there. Just saving and reloading will save the day in this case.

command line when you get blank tracks sometimes gives you messages like this in abundance:

```

called Track::getTCO( 4 ), but TCO 4 doesn't exist
QObject: Cannot create children for a parent that is in a different thread.
(Parent is InstrumentTrack(0x7f3998693420), parent's thread is QThread(0x8f1cd0), current thread is QThread(0xbd8da0)
```

![beattracks](https://cloud.githubusercontent.com/assets/6368949/17342887/82d0dd24-58fb-11e6-9c5b-b27388eca6e7.png)

![blanktracks](https://cloud.githubusercontent.com/assets/6368949/17342905/8f895fd2-58fb-11e6-823b-2e223abc0757.png)

**Comment by musikBear** [comment]
This is in linux, right?

**Comment by zonkmachine** [comment]
> This is in linux, right?

Right, I can't test the others. It's already in 1.0.3

**Comment by zonkmachine** [comment]
It looks like this was fixed in #2981 It hasn't reappeared since. Closing.
